scripts/build_docker_image.sh

```diff
@@ -21,12 +21,12 @@ if [[ "${VM_ID}" != "${DEFAULT_VM_ID}"  ]]; then
 fi
 
 # Default to the release image. Will need to be overridden when testing against unreleased versions.
-AVALANCHE_NODE_IMAGE=${AVALANCHE_NODE_IMAGE:-"avaplatform/avalanchego:${AVALANCHE_VERSION}"}
+AVALANCHEGO_NODE_IMAGE=${AVALANCHEGO_NODE_IMAGE:-"avaplatform/avalanchego:${AVALANCHE_VERSION}"}
```

Comment:
```suggestion
AVALANCHEGO_NODE_IMAGE=${AVALANCHEGO_NODE_IMAGE:-"avaplatform/avalanchego:${AVALANCHE_VERSION::8}"}
```
@ceyonur if you like it this way feel free to commit it